src/lib/config.ts

```diff
@@ -486,6 +487,16 @@ export const WEBRESOURCES_CLOUDFRONT_HOST = optionalVar(
 	'WEBRESOURCES_CLOUDFRONT_HOST',
 );
 
+export const DISABLED_SCHEDULED_JOBS = new Set(
+	arrayVar('DISABLED_SCHEDULED_JOBS') ?? [],
+);
+export const disableScheduledJob = (jobName: string) => {
```

Comment:
I guess it would be useful to also have a way to disable all the jobs, but we can do that later if needed :+1: